pkg/db/db_helpers_test.go

```diff
@@ -54,7 +54,7 @@ func getDBConfig(t *testing.T) dbConfig {
 	if dbPool != nil {
 		const password = "secret"
 		// pulls an image, creates a container based on it and runs it
-		resource, err := dbPool.Run("postgres", "13", []string{"POSTGRES_PASSWORD=" + password})
```

Comment:
Was this change intentional?

Reply:
Yup, this is the version we have in RDS. I noticed the mismatch and corrected it, but only mentioned that in a commit message.

Reply:
:+1: